doc/source/skips/1-governance.rst

```diff
@@ -145,16 +146,18 @@ are made according to the following rules:
   sentence (but no change of the scikit-image.org landing page or the “about”
   page), require approval by a core developer *and* no disagreement or requested
   changes by a core developer on the issue or pull request page (lazy
-  consensus). Core developers are expected to give “reasonable time” to others
-  to give their opinion on the pull request if they’re not confident others
-  would agree.
+  consensus). Core developers are expected to wait one or two days for others to give
+  their opinion if they’re not confident that others would agree.
 
-- **Code changes and major documentation changes** require agreement by *two*
-  core developers *and* no disagreement or requested changes by a core developer
-  on the issue or pull-request page (lazy consensus).
+- **Code and major documentation changes, and changes to the API** require agreement by
+  *two* core developers *and* no disagreement or requested changes by a core developer
+  on the issue or pull request page (lazy consensus). In the case of disagreement or
+  requested changes, core developers are expected to wait about a week for others to
+  give their opinion if they’re not confident that others would agree.
 
 - **Changes to the API principles** require a :ref:`SKIP <skip>` and follow the
-  decision-making process outlined above.
+  decision-making process outlined above. But, in this case, the objection period
```

Comment:
A month seems very long. Do we need that much time?

Reply:
Apparently, we do! It's been way over a month since three core devs have approved this PR, I'd love to merge it now, but I admit I'd prefer having @stefanv's approval as well.

Reply:
OK to go ahead with this PR as-is, as to not hold it up, but I do suggest we pull it back to two weeks to give the dev team license to move more quickly.